notebooks/214-vehicle-detection-and-recognition/214-vehicle-detection-and-recognition.ipynb

```diff
@@ -0,0 +1,545 @@
+   "source": [
+    "import os\n",
+    "from typing import Tuple\n",
+    "\n",
+    "import cv2\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "from openvino.inference_engine import IECore"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "448e7312",
+   "metadata": {},
+   "source": [
+    "# Download Models\n",
+    "\n",
+    "We need to download pretrained models to continue our progress.We use `omz_downloader`, a command-line tool installed by `openvino-dev` package.\n",
+    "\n",
+    "> Note: If you want to change the model,you need to modify the model name.If you want to change the precision,you need to modify the precision value. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "94f2c2be",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Directory where model will be downloaded\n",
+    "base_model_dir = \"model\"\n",
+    "# Model name as named in Open Model Zoo\n",
+    "detection_model_name = \"vehicle-detection-0200\"\n",
+    "recognition_model_name = \"vehicle-attributes-recognition-barrier-0039\"\n",
+    "# Selected precision (FP32, FP16, FP16-INT8)\n",
+    "precision = \"FP32\"\n",
+    "\n",
+    "# Check the model exists \n",
+    "detection_model_path = (\n",
+    "    f\"model/intel/{detection_model_name}/{precision}/{detection_model_name}.xml\"\n",
+    ")\n",
+    "recognition_model_path = (\n",
+    "    f\"model/intel/{recognition_model_name}/{precision}/{recognition_model_name}.xml\"\n",
+    ")\n",
+    "\n",
+    "# Download detection model\n",
+    "if not os.path.exists(detection_model_path):\n",
+    "    download_command = f\"omz_downloader \" \\\n",
+    "                       f\"--name {detection_model_name} \" \\\n",
+    "                       f\"--precision {precision} \" \\\n",
+    "                       f\"--output_dir {base_model_dir}\"\n",
+    "    ! $download_command\n",
+    "# Download recognition model\n",
+    "if not os.path.exists(recognition_model_path):\n",
+    "    download_command = f\"omz_downloader \" \\\n",
+    "                       f\"--name {recognition_model_name} \" \\\n",
+    "                       f\"--precision {precision} \" \\\n",
+    "                       f\"--output_dir {base_model_dir}\"\n",
+    "    ! $download_command"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "61c678fb",
+   "metadata": {},
+   "source": [
+    "# Load Models\n",
+    "\n",
+    "In this notebook,we will need detection model and recognition model.When we download models,we need to initialize inference engine(IECore),and use `read_network` to read network architecture and weights from *.xml and *.bin files.Then,we load the network on the \"CPU\" using `load_network`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "41d874b5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Initialize inference engine\n",
+    "ie_core = IECore()\n",
```

Comment:
You're using the old OpenVINO API. It was changed just after you opened this PR. If you can please update to the new API.